extensions/sftp/client/SFTPClient.cpp

```diff
@@ -83,7 +83,7 @@ static const char* sftp_strerror(unsigned long err) {  // NOLINT(runtime/int) un
   }
 }
 
-static SFTPError libssh2_sftp_error_to_sftp_error(unsigned long libssh2_sftp_error) {  // NOLINT(runtime/int) unsigned long comes from libssh2 API
+static SFTPError libssh2_sftp_error_to_sftp_error(unsigned long libssh2_sftp_error) {  // NOLINT(runtime/int,google-runtime-int) unsigned long comes from libssh2 API
```

Comment:
We can disable one of these, so we only need one NOLINT for the same rule

Reply:
There were instances for both checks where only one of them gave a warning for this issue, so I kept both of them.